packages/devtools/lib/src/utils.dart

```diff
@@ -57,13 +57,23 @@ String printMb(num bytes, [int fractionDigits = 1]) {
   return (bytes / (1024 * 1024)).toStringAsFixed(fractionDigits);
 }
 
-String microsAsMsText(num micros,
-    {bool includeUnit = true, int fractionDigits = 1}) {
-  return msAsText(micros / 1000, includeUnit: includeUnit);
+String microsAsMsText(
+  num micros, {
+  bool includeUnit = true,
+  int fractionDigits = 1,
+}) {
+  return msAsText(
+    micros / 1000,
+    includeUnit: includeUnit,
+    fractionDigits: fractionDigits,
+  );
 }
 
-String msAsText(num milliseconds,
-    {bool includeUnit = true, int fractionDigits = 1}) {
+String msAsText(
```

Comment:
Maybe change these to one method
formatTimeInMilliseconds({
  num microseconds,
  num milliseconds, ...
}) {
  assert(microseconds == null || milliseconds == null);
  if (millliseconds == null) {
     assert(microseconds != null)
     milliseconds = micros / 100;
  }
  Alternately you could use the Duration class to represent times and pass in Duration objects which would also make this a bit cleaner to compute.

Reply:
done.